scripts/zones/Temple_of_Uggalepih/mobs/Cook_Fulberry.lua

```diff
@@ -0,0 +1,19 @@
+-----------------------------------
+-- Area: Temple of Uggalepih
+--   NM: Cook Fulberry
+-----------------------------------
+mixins = { require('scripts/mixins/job_special') }
+-----------------------------------
+---@type TMobEntity
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    xi.mix.jobSpecial.config(mob, {
+        specials =
+        {
+            { id = xi.jsa.ASTRAL_FLOW },
+        },
+    })
```

Comment:
This isn't required. It's done by loading the mixin above like you've done.

Reply:
Ahh, gotcha. Thanks for the explanation!